packages/blocks/news/6536.internal

```diff
@@ -0,0 +1 @@
+Update RAC to latest. @sneridagh
```

Comment:
This is not what changed in `blocks` :D it's the tsconfig

Reply:
Fixed!